services-worker/src/test/resources/edu/unc/lib/dl/cdr/services/techmd/fitsOutputMultipleLineBreaks.xml

```diff
@@ -1,5 +1,3 @@
-2015-06-13 06:19:59,410 ERROR [main] ToolBelt:79 - Thread 1 error initializing edu.harvard.hul.ois.fits.tools.exiftool.Exiftool: edu.harvard.hul.ois.fits.exceptions.FitsToolCLIException  Message: Error calling external command line routine (Cannot run program "which": java.io.IOException: error=12, Cannot allocate memory)
```

Comment:
I reverted this file, @bbpennel . Sorry, I didn't realize it was important; Eclipse was complaining that the line wasn't allowed in prolog.